### PR TITLE
Add Process for Changes to Code of Conduct

### PR DESCRIPTION
**Pull request by liam-middlebrook**
Check one:
- [x] Semantic Change: something about the meaning of the text is different
- [ ] Non-semantic Change: Spelling, grammar, or formatting changes.

Summary of change(s):

Defines the Code of Conduct and the process by which changes to the Code
of Conduct can be make.

### CODE REVIEW

File: bylaws.tex [hunk 1]
```diff
@@ -358,4 +358,15 @@
...
+\bsection{Code of Conduct}
+The Code of Conduct located at \href{https://github.com/ComputerScienceHouse/CodeOfConduct} is the canonical Code of Conduct for Computer Science House accounts.
+\\* \\*
+Members are bound to the Code of Conduct revision that they sign when initially creating their account. Members may sign a more recent revision of the Code of Conduct to update their agreement.
+\bsubsection{Changes}
+The following process is used for making changes to the Code of Conduct document.
+\begin{enumerate}
+	\item A Root Type Person drafts a change to the Code of Conduct and makes the difference file of the change as well as the summary of the change publicly available.
+	\item The change is proposed at a House Meeting and is discussed in the same House Meeting.
+	\item The final proposal is presented at the next House Meeting and ballots are distrubuted for a Balloted House vote as described in \ref{Balloted Vote}.
```
Comment (stevenmirabito) [inline bylaws.tex]: Typo, should be "distributed"

File: bylaws.tex [hunk 2]
```diff
@@ -358,4 +358,15 @@
...
 
+\bsection{Code of Conduct}
+The Code of Conduct located at \href{https://github.com/ComputerScienceHouse/CodeOfConduct} is the canonical Code of Conduct for Computer Science House accounts.
+\\* \\*
+Members are bound to the Code of Conduct revision that they sign when initially creating their account. Members may sign a more recent revision of the Code of Conduct to update their agreement.
+\bsubsection{Changes}
+The following process is used for making changes to the Code of Conduct document.
+\begin{enumerate}
+	\item A Root Type Person drafts a change to the Code of Conduct and makes the difference file of the change as well as the summary of the change publicly available.
+	\item The change is proposed at a House Meeting and is discussed in the same House Meeting.
```
Comment (stevenmirabito) [inline bylaws.tex]: "discussed in" should be "discussed at" to keep it consistent.

File: bylaws.tex [hunk 3]
```diff
@@ -358,4 +358,14 @@
...
 \end{enumerate}
 
+\bsection{Code of Conduct}
+The Code of Conduct located at \href{https://github.com/ComputerScienceHouse/CodeOfConduct} is the canonical Code of Conduct for Computer Science House accounts.
+\\* \\*
+Members are bound to the Code of Conduct revision that they sign when initially creating their account. Members may sign a more recent revision of the Code of Conduct to update their agreement.
+\bsubsection{Changes}
+The following process is used for making changes to the Code of Conduct document.
+\begin{enumerate}
+	\item A Root Type Person drafts a change to the Code of Conduct and makes the difference file of the change as well as the summary of the change publicly available.
```
Comment (harlanhaskins) [inline bylaws.tex]: The 'as well as' here makes this sentence awkward.

A Root Type Person drafts a change to the Code of Conduct and makes publicly available both the summary and difference file of the change.

File: bylaws.tex [hunk 4]
```diff
@@ -358,4 +358,14 @@
...
+	\item The change is proposed at a House Meeting and is discussed at the same House Meeting.
+	\item The final proposal is presented at the next House Meeting and ballots are distributed for a Balloted House vote as described in \ref{Balloted Vote}.  \end{enumerate}
```
Comment (harlanhaskins) [inline bylaws.tex]: Newline before the `\end`

**Comment by liam-middlebrook** [comment]
Passed 32-1
I'll merge this later tonight after a quick rebase.

**Pull request merged**